update_all.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
```

Comment:
Please remove this script.  We already have `bin/utils/ensure-up-to-date`.

Reply:
was added accidentally. removed